projects/extension/src/background/index.ts

```diff
@@ -164,6 +169,13 @@ window.uiInterface = {
       return []
     }
   },
+  get integratedChains() {
+    const s: string[] = []
+    for (let value of wellKnownChains.values()) {
+      s.push(value[0])
+    }
+    return s
+  },
```

Comment:
You don't actually need this.
Right now we intentionally don't report the well-known chains in `chains`, but it's more appropriate to simply report that in `get chains()` rather than add a separate method.